src/www/ui/api/index.php

```diff
@@ -111,6 +111,7 @@ function (){
     $this->post('', UploadController::class . ':postUpload');
     $this->get('/{id:\\d+}/summary', UploadController::class . ':getUploadSummary');
     $this->get('/{id:\\d+}/licenses', UploadController::class . ':getUploadLicenses');
+    $this->options('/{params:.*}', AuthController::class . ':optionsVerification');
```

Comment:
A much simpler way will to add something like following just before the middleware configuration:
```php
///////////////////////CORS OPTIONS////////////////
$app->options(VERSION_1 . '{routes:.+}', AuthController::class . ':optionsVerification');
```